cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/PersonPatch.java

```diff
@@ -0,0 +1,71 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.ALWAYS;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.Objects;
+import java.util.Optional;
+import no.unit.nva.cristin.person.update.PersonPatchSerializer;
+import nva.commons.core.JacocoGenerated;
+
+@JsonSerialize(using = PersonPatchSerializer.class)
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+@JsonInclude(ALWAYS)
+@JacocoGenerated
+public class PersonPatch {
+
+    private transient Optional<String> orcid;
+    private transient Optional<String> firstName;
+    private transient Optional<String> lastName;
+    private transient Optional<String> preferredFirstName;
+    private transient Optional<String> preferredLastName;
+    private transient Optional<Boolean> reserved;
```

Comment:
I understand why this is done — other reviewers may not, probably worth a comment in the code

Reply:
Will do in next PR